x-pack/plugins/profiling/server/lib/setup/get_has_setup_privileges.ts

```diff
@@ -0,0 +1,34 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { KibanaRequest } from '@kbn/core/server';
+import { INTEGRATIONS_PLUGIN_ID, PLUGIN_ID as FLEET_PLUGIN_ID } from '@kbn/fleet-plugin/common';
+import { ProfilingPluginStartDeps } from '../../types';
+
+export async function getHasSetupPrivileges({
+  securityPluginStart,
+  request,
+}: {
+  securityPluginStart: NonNullable<ProfilingPluginStartDeps['security']>;
+  request: KibanaRequest;
+}) {
+  const { hasAllRequested } = await securityPluginStart.authz
+    .checkPrivilegesWithRequest(request)
+    .globally({
+      elasticsearch: {
+        cluster: ['manage', 'monitor'],
```

Comment:
According to the [docs](https://www.elastic.co/guide/en/elasticsearch/reference/current/security-privileges.html), `manage` is a superset of `monitor`. Did your testing reveal that both of these were actually required?

![Larry Gregory 2023-10-23 at 07 49 47](https://github.com/elastic/kibana/assets/3493255/ce92beea-8aae-4524-9d73-a3064050db19)

Reply:
Yes, both are needed. 
- `Manage` is used to update the cluster settings enabling profiling index management and bucket settings.
- `Monitor` is needed by the ES Profiling status API.

Reply:
correct!

Reply:
Correct